src/west/config.py

```diff
@@ -22,13 +24,33 @@
 config = configparser.ConfigParser()
 
 
-def read_config():
+class ConfigFile(Enum):
+    ALL = 0
+    if platform.system() == 'Linux':
+        SYSTEM = '/etc/westconfig'
+    elif platform.system() == 'Darwin':  # Mac OS
```

Comment:
I believe it's `darwin`, lower case, see: https://github.com/zephyrproject-rtos/west/commit/22b395c75ca5144864330bf723e8f8b1b881353d

Reply:
Code is actually just moved from here: https://github.com/zephyrproject-rtos/west/blob/master/src/west/config.py#L73
so didn't want to change the casing, as I believe existing code to be working on MacOS.

Reply:
`platform.system()` should return the same as `uname`

Reply:
Hmm you are right, it returns `Darwin`. Strange that the change in `tox.ini` works then.

Reply:
maybe 'darwin' is tox internal to tell it to report `Darwin` among setting up other things as well, to emulate that system.
So that it is more than just returning the name.

Reply:
I see that `platform.system()` return `Linux` with a capital `L` too, so tox must be normalizing the case. No issues from me here then.